sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -47,7 +47,7 @@
 /**
  * @brief   Determine length of array in elements
  */
-#define _ARRAY_LEN(x)                   (sizeof(x) / sizeof(x[0]))
+#define _ARRAY_LEN(x)                   ARRAY_SIZE(x)
```

Comment:
Why not remove `_ARRAY_LEN` and replace its use with `ARRAY_SIZE`?

Reply:
Good catch!